custom_components/zonneplan_one/const.py

```diff
@@ -107,94 +107,94 @@ class ZonneplanSensorEntityDescription(SensorEntityDescription):
             icon="mdi:message-text-outline",
             entity_registry_enabled_default=True,
         ),
-        "forcast_tariff_1": ZonneplanSensorEntityDescription(
+        "forecast_tariff_1": ZonneplanSensorEntityDescription(
```

Comment:
Please keep these keys as they are. That would be a breaking change

Reply:
All of it is a breaking change....

Reply:
If the key is kept then it will not break for existing installs.

The `entity id` wil not change because the sensor in HA is matched on the identifyer and this key is used for this. When the sensor is new in HA the label is used to create a `entity id` and that will not change when the label changes

Reply:
That's exactly what you _should_ do. We shouldn't just change the label, also the entity_id should change. I really don't see why you should have something linger around that is wrong? 

The enity_id is used in many places, especially when not using the UI to create dashboards and/or when working with templates. Your suggestion will just keep the wrong spelling for all those cases, now and in future.

Reply:
Misschien samen met #45 - om dat te fixen moet de component sowieso opnieuw geladen worden. Of heb je daar misschien een slimmere manier voor?

Reply:
give it some more thought, I see no reason to make it breaking by changing a key that is only used for internals.

Reply:
changing the name is not issue, new installs will get a correct entity id. For existing installs users can change the entity id them self when they have an issue with the wrong spelling :)

Reply:
I'll have a go at it prob tomorrow - busy day today